lib/structures/GuildChannel.js

```diff
@@ -49,6 +49,7 @@ class GuildChannel extends Channel {
         this.position = data.position !== undefined ? data.position : this.position;
         this.bitrate = data.bitrate !== undefined ? data.bitrate : this.bitrate;
         this.userLimit = data.user_limit !== undefined ? data.user_limit : this.userLimit;
+        this.nsfw = data.nsfw !== undefined ? data.nsfw : false;
```

Comment:
this.nsfw = this.type !== 2 && ((this.name.length === 4 ? this.name === "nsfw" : this.name.startsWith("nsfw-")) || data.nsfw);